--- origsrc/libcanberra-0.26/src/Makefile.am	2010-12-14 00:56:46.350351800 -0600
+++ src/libcanberra-0.26/src/Makefile.am	2010-12-14 01:52:56.115388000 -0600
@@ -316,8 +316,8 @@ libcanberra_gtk3_module_la_LDFLAGS = \
 install-exec-hook:
 	$(MKDIR_P) -m 755 $(DESTDIR)$(gtk3moduledir)
 	( cd $(DESTDIR)$(gtk3moduledir) && \
-		rm -f libcanberra-gtk-module.so && \
-	        $(LN_S) libcanberra-gtk3-module.so libcanberra-gtk-module.so )
+		rm -f cygcanberra-gtk-module.dll && \
+	        $(LN_S) cygcanberra-gtk3-module.dll cygcanberra-gtk-module.dll )
 
 endif
 
