--- origsrc/libcanberra-0.26/src/Makefile.am	2010-12-14 00:56:46.350351800 -0600
+++ src/libcanberra-0.26/src/Makefile.am	2010-12-14 01:52:56.115388000 -0600
@@ -353,24 +353,24 @@ libcanberra_gtk_module_la_LDFLAGS = \
 
 endif
 
-if HAVE_GTK3
+if HAVE_GTK
 
 canberra_gtk_play_LDADD = \
-	$(GTK3_LIBS) \
+	$(GTK_LIBS) \
 	libcanberra.la \
-	libcanberra-gtk3.la
+	libcanberra-gtk.la
 canberra_gtk_play_CFLAGS = \
-	$(GTK3_CFLAGS)
+	$(GTK_CFLAGS)
 
 else
-if HAVE_GTK
+if HAVE_GTK3
 
 canberra_gtk_play_LDADD = \
-	$(GTK_LIBS) \
+	$(GTK3_LIBS) \
 	libcanberra.la \
-	libcanberra-gtk.la
+	libcanberra-gtk3.la
 canberra_gtk_play_CFLAGS = \
-	$(GTK_CFLAGS)
+	$(GTK3_CFLAGS)
 
 endif
 endif
