--- origsrc/libcanberra-0.4/src/Makefile.am	2008-07-25 10:15:50.000000000 -0500
+++ src/libcanberra-0.4/src/Makefile.am	2008-08-13 00:52:08.385588800 -0500
@@ -56,6 +56,7 @@
 	$(VORBIS_LIBS)
 libcanberra_la_LDFLAGS = \
 	-export-dynamic \
+	-no-undefined \
 	-version-info $(LIBCANBERRA_VERSION_INFO) \
 	-Wl,-version-script=$(srcdir)/map-file
 
@@ -87,7 +88,7 @@
 libcanberra_multi_la_LIBADD = \
 	libcanberra.la
 libcanberra_multi_la_LDFLAGS = \
-	-avoid-version -module -export-dynamic
+	-avoid-version -module -no-undefined -export-dynamic
 endif
 
 if HAVE_PULSE
@@ -120,7 +121,7 @@
 	$(PULSE_LIBS) \
 	libcanberra.la
 libcanberra_pulse_la_LDFLAGS = \
-	-avoid-version -module -export-dynamic
+	-avoid-version -module -no-undefined -export-dynamic
 
 endif
 endif
@@ -184,7 +185,7 @@
 libcanberra_null_la_LIBADD = \
 	libcanberra.la
 libcanberra_null_la_LDFLAGS = \
-	-avoid-version -module -export-dynamic
+	-avoid-version -module -no-undefined -export-dynamic
 endif
 endif
 
@@ -211,7 +212,7 @@
 	$(GTK_LIBS) \
 	libcanberra.la
 libcanberra_gtk_la_LDFLAGS = \
-	-export-dynamic -version-info $(LIBCANBERRA_GTK_VERSION_INFO)
+	-export-dynamic -version-info $(LIBCANBERRA_GTK_VERSION_INFO) -no-undefined
 
 libcanberra_gtk_module_la_SOURCES = \
 	canberra-gtk-module.c
@@ -222,7 +223,7 @@
 	libcanberra.la \
 	libcanberra-gtk.la
 libcanberra_gtk_module_la_LDFLAGS = \
-	-avoid-version -module -export-dynamic
+	-avoid-version -module -no-undefined -export-dynamic
 
 canberra_gtk_play_SOURCES = \
 	canberra-gtk-play.c
